resources/key.rb

```diff
@@ -0,0 +1,103 @@
+=begin
+#<
+Manage rundeck key storage through rundeck api
+
+@action create Create a key in storage
+@action delete Delete key from storage
+
+@section Examples
+
+    rundeck_server_key 'mykey' do
+      type 'private'
+      content '---START.....'
+      endpoint 'ttp://127.0.0.1:4440'
+      api_token 'myticket''
+      action :create
+    end
+#>
+=end
+
+# <> @property path to key
+property :key,    kind_of: String, name_property:  true
+# <> @property tyoe of key. Can be 'private', 'public'
```

Comment:
optional: typo on `tyoe`

Reply:
I think there is also a way to specify the possible values that a property can take

Reply:
Fixed